tests/utils.py

```diff
@@ -42,8 +42,8 @@ def generate_regression_model(features_shape, output_shape=1):
     model.add(Dense(4, activation='relu'))
     model.add(Dense(4, activation='relu'))
     model.add(Dense(output_shape))
-    model.compile(loss='mean_absolute_error',
-                  optimizer='sgd')
+    model.compile(loss='mean_absolute_error', optimizer='sgd',
+                  metrics=['accuracy'])
```

Comment:
I don't believe that having metrics for test is useful. What do you think ? :)

Reply:
You're right

Reply:
Having metrics for test is useful for causal fidelity for tabular data, otherwise you'll have an error when you test another metric than _loss_ (e.g. _accuracy_):
>        assert metric == "loss" or metric in self.model.metrics_names